File: frontend/angular/src/app/core/dtos/customers/customer.dto.ts
export interface CustomerDto {
  id: number;
  companyName: string;
  phone: string;
  email: string;
  address: string;
}

export interface CreateCustomerRequestDto {
  companyName: string;
  phone: string;
  email: string;
  address: string;
}

export interface UpdateCustomerRequestDto {
  companyName: string;
  phone: string;
  email: string;
  address: string;
}

File: backend/CncApp/CncApp.Application/Dtos/Customers/CustomerDto.cs
namespace CncApp.Application.Dtos.Customers;

public class CustomerDto
{
    public int Id { get; set; }

    public string CompanyName { get; set; } = string.Empty;

    public string Phone { get; set; } = string.Empty;

    public string Email { get; set; } = string.Empty;

    public string Address { get; set; } = string.Empty;
}
